bcipy/display/paradigm/vep/display.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 from psychopy import visual, core
 from bcipy.helpers.clock import Clock
-from bcipy.helpers.task import SPACE_CHAR
```

Comment:
We needed to move those things so that our evaluation scripts didn't need to import psychopy, which was preventing them from running headless with pygame. I believe psychopy was being imported by whatever contained alphabet(), SPACE_CHAR, and BACKSPACE_CHAR previously. Is the language module not the primary module that uses those constants and alphabet()?

Reply:
Created a specific helper for these imports